cgmes/cgmes-extensions/src/main/java/com/powsybl/cgmes/extensions/CgmesTapChangerAdder.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.extensions;
+
+/**
+ * @author Miora Vedelago <miora.ralambotiana at rte-france.com>
+ */
+public interface CgmesTapChangerAdder {
+
+    CgmesTapChangerAdder setId(String id);
+
+    CgmesTapChangerAdder setCombinedTapChangerId(String combinedTapChangerId);
+
+    CgmesTapChangerAdder setType(String type);
+
+    CgmesTapChangerAdder setHiddenStatus(boolean hidden);
+
+    CgmesTapChangerAdder setStep(int step);
```

Comment:
Why do we need to store the step here?

Reply:
It is the tapPosition of the hidden tapChanger.